src/app/shared/issue-tables/IssuesDataTable.ts

```diff
@@ -63,7 +65,25 @@ export class IssuesDataTable extends DataSource<Issue> implements FilterableSour
       .pipe(
         // maps each change in display value to new issue ordering or filtering
         map(() => {
+          // check if the current group is in the filter
+          const groupFilterAsGithubUser = this.filter.assignees.map((selectedAssignee) => {
+            return GithubUser.fromUsername(selectedAssignee);
+          });
+          const groupFilterAsMilestone = this.filter.milestones.map((selectedMilestone) => {
+            return Milestone.fromTitle(selectedMilestone);
+          });
+
+          const isGroupInFilter =
+            groupFilterAsGithubUser.filter((githubUser) => this.group?.equals(githubUser)).length !== 0 ||
+            groupFilterAsMilestone.filter((milestone) => this.group?.equals(milestone)).length !== 0;
+
+          if (!isGroupInFilter) {
+            this.count = 0;
+            return [];
+          }
+
```

Comment:
I'm not entirely convinced this is the best solution because it feels a bit hacky. However, I don't see an easier way to achieve this without changing the storage type of filter.

One way to improve this would be to encapsulate the logic into a separate function, like `isGroupInFilter(group: Group, filter: Filter)`. This would make the `map` more readable and less hacky.